### DISCUSSION TITLE
There's a infinite-spell hack? it is visible for others? If these are a yes, how can i obtain it?

### DISCUSSION THREAD

**Discussion by SoyJoshuaDeCa** [thatskymod/Sky-CotL-Scripts#76] [Q&A]
Is there a infinite-spell hack with game guardian? if there is, how can i get it? i saw ones with GameGuardian, but these are outdated and im not sure. If there are other methods, please lemme know. i was searching for a infinite spell duration for the Violin.

**Comment by VK-TCheeseCake** [comment]
I dont think we can anymore since sky reverted it and not allow us to make it inf anymore

**Comment by thatskymod** [comment]
Similar to what is outlined in [#Outfit Visibility](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/DOCS.md#outfit-visibility), the reason spells are not visible to others beyond their standard duration is that the validity period is now managed server-side rather than locally. This means that although you may be able to freeze the spell timer on your end, this modification will not carry over to the backend, and consequently, to other players.
